consensus/src/sync/live/block_queue/mod.rs

```diff
@@ -272,6 +272,10 @@ impl<N: Network> BlockQueue<N> {
     /// Requests missing blocks.
     /// If a block locator is given, it is inserted at the beginning of the full block locator list.
     /// The list contains all blocks from the head until the last macro block.
+    ///
+    /// `pubsub_id` specifies the gossipsub ID that of the gossip that led to
```

Comment:
```suggestion
    /// `pubsub_id` specifies the gossipsub ID that of the peer that led to
```

Reply:
Maybe the "gossipsub peer ID of the gossip message that led to…"? It's the message leading to the request, not the peer.

Reply:
Ah, that makes sense

Reply:
Fixed.